Healthy/Classes/ReusableViews/CheckboxButton.swift

```diff
@@ -0,0 +1,30 @@
+import UIKit
+
+class CheckboxButton: UIButton {
+
+    // Images
+    let checkedImage = UIImage.icCheckboxSelected
+    let uncheckedImage = UIImage.icCheckboxNotSelected
+
+    // Bool property
+    var isChecked: Bool = false {
+        didSet {
+            if isChecked == true {
+                self.setImage(checkedImage, for: UIControl.State.normal)
+            } else {
+                self.setImage(uncheckedImage, for: UIControl.State.normal)
+            }
+        }
+    }
+
+    override func awakeFromNib() {
```

Comment:
Make sure you call super to avoid any issues.
```suggestion
    override func awakeFromNib() {
      super.awakeFromNib()
```